Detectors/FOCAL/simulation/src/Detector.cxx

```diff
@@ -418,9 +485,31 @@ void Detector::addAlignableVolumesHCAL() const
   }
 }
 
+//____________________________________________________________________________
+void Detector::addAlignableVolumesECAL() const
+{
+  TString vpsector = "/cave_1/caveRB24_1/FOCAL_1/ECAL_1";
+  TString snsector = "FOCAL/ECAL";
```

Comment:
Please avoid using classes from ROOT unless needed. TString in this case should be converted to std::string